src/draw/lv_draw_vector.h

```diff
@@ -86,6 +86,11 @@ typedef enum {
     LV_VECTOR_GRADIENT_STYLE_RADIAL,
 } lv_vector_gradient_style_t;
 
+typedef enum {
+    LV_VECTOR_FILL_UTILES_OBJECT_BOUDING_BOX = 0,
+    LV_VECTOR_FILL_UTILES_USER_SPACE_ON_USE,
```

Comment:
`utils` is not suitable name, how about using `Units` like SVG, such as: LV_VECTOR_FILL_UNITS_OBJECT_BOUDING_BOX